moma_demos/piloting_demo/launch/perception.launch

```diff
@@ -1,33 +1,38 @@
 <?xml version="1.0"?>
 <launch>
-  <arg name="play_bag"     default="false"/>
-  <arg name="bag_file"     default="$(env HOME)/storage/bags/2022-01-26/2022-01-26-11-11-18_valve_depth_throttled.bag"/>
-  <arg name="base_frame"   default="panda_link0"/>
-  <arg name="camera_frame" default="hand_eye_color_optical_frame"/>
-  <arg name="image_topic"  default="/hand_eye/color/image_raw_throttle"/>
-  <arg name="depth_topic"  default="/hand_eye/depth/image_rect_raw_throttle"/>
-  <arg name="calibration_topic"  default="/camera_info"/>
+  <arg name="play_bag"          default="false"/>
+  <arg name="bag_file"          default="$(env HOME)/storage/bags/2022-01-26/2022-01-26-11-11-18_valve_depth_throttled.bag"/>
+  <arg name="base_frame"        default="panda_link0"/>
+  <arg name="camera_frame"      default="hand_eye_color_optical_frame"/>
+  <arg name="image_topic"       default="/hand_eye/color/image_raw_throttle"/>
+  <arg name="depth_topic"       default="/hand_eye/depth/image_rect_raw_throttle"/>
+  <arg name="calibration_topic" default="/camera_info"/>
+  <arg name="calibration_file"  default="$(env HOME)/calibration/intrinsics.yaml"/>
+  <arg name="custom_intrinsics" default="false"/>
+  <arg name="rviz"              default="false"/>
 
   <!-- Custom intrinsics publishers -->
   <arg name="custom_intrinsics_file" default="$(env HOME)/calibration/camera_calibration.yaml" />
-  <node name="camera_intrinsics_publisher" pkg="moma_sensor_tools" type="yaml_to_camera_info_publisher.py" args="$(arg custom_intrinsics_file)" output="screen"/>
+  <node name="camera_intrinsics_publisher" pkg="moma_sensor_tools" type="yaml_to_camera_info_publisher.py" 
+    args="$(arg custom_intrinsics_file)" output="screen" if="$(arg custom_intrinsics)"/>
 
   <include file="$(find object_keypoints_ros)/launch/keypoints.launch">
     <arg name="load_bag_filepath" default="false"/>
-    <arg name="calibration_file"  default="$(env HOME)/calibration/intrinsics.yaml"/>
+    <arg name="calibration_file"  default="$(arg calibration_file)"/>
     <arg name="image_topic"       default="$(arg image_topic)"/>
     <arg name="depth_topic"       default="$(arg depth_topic)"/>
     <arg name="calibration_topic" default="$(arg calibration_topic)"/>
     <arg name="base_frame"        default="$(arg base_frame)"/>
     <arg name="camera_frame"      default="$(arg camera_frame)"/>
-    <arg name="model_name"        default="valve_depth"/>
+    <arg name="model_name"        default="valve"/>
```

Comment:
@grizzi Note that I updated `object_keypoints_ros`.
I think it makes more sense to have a general "valve" name and in the end also rename the "valve_depth.pt" to "valve.pt". This way we have a unified naming, also for "valve.json" for the keypoints, etc.